week1/My-own-modified/index.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
+const http = require("http");
+const server = http.createServer((req, res) => {
+  console.log("Creating Server");
+});
+let state = 10;
+let original = 10;
+server.on("request", (req, res) => {
+  function responseWrite(val) {
+    res.setHeader("content-type", "text/html");
+    res.write(`<h1>${val}</h1>`);
+  }
+  console.log(req.method, req.url);
+  switch (req.url) {
+    case "/state":
+      responseWrite(state)
+      break;
+    case "/add":
+      responseWrite(state += 1);
+      break;
+    case "/remove":
+      responseWrite(state -= 1);
+      break;
+    case "/reset":
+      responseWrite(original);
```

Comment:
I like the way you structured your refactoring, but line 25 (here) doesn't reset the state to the original, it just prints the original.